lego/apps/events/templates/events/email/admin_unregistration.html

```diff
@@ -0,0 +1,40 @@
+{% extends "email/base.html" %}
+
+{% block alert %}
+
+    <tr>
+        <td class="alert alert-bad">
+            Fjernet fra event
+        </td>
+    </tr>
+
+{% endblock %}
+
+{% block content %}
+
+    <tr>
+        <td class="content-block">
+            Hei {{ name }}!
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            En administrator har fjernet deg fra arrangementet {{ event }} på grunn av:
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            <i>{{ reason }}</i>
+        </td>
+    </tr>
+
+    <tr>
+        <td class="content-block">
+            Ta kontakt med {{ creator }} dersom du skulle være uenig i dette.
```

Comment:
Where does `creator`come from?

Reply:
Here: https://github.com/webkom/lego/pull/974/files#diff-04d0a38218aa36f65d77e821e0c156e1R128

But forgot to add it in the Notification